services/cases-api/src/helpers/statusCheckCondition.js

```diff
@@ -12,21 +12,21 @@ import {
 } from '../libs/constants';
 
 function isSignaturePending({ answers, people }) {
-  return (
-    answers && isEncrypted(answers) && hasApplicantSigned(people) && !hasCoApplicantsSigned(people)
-  );
+  const isCoApplicantSignPending = !isCoApplicantSigned(people);
+  return answers && isEncrypted(answers) && isApplicantSigned(people) && isCoApplicantSignPending;
```

Comment:
If we are sure that `answers` is always an object or array then this should be fine:

```js
return isEncrypted(answers) && isApplicantSigned(people) && isCoApplicantSignPending;
```